calico/felix/fetcd.py

```diff
@@ -191,7 +269,7 @@ class _EtcdWatcher(gevent.Greenlet):
 
     def __init__(self, config, hosts_ipset):
         super(_EtcdWatcher, self).__init__()
-        self.config = config
+        self._config = config
```

Comment:
You're renaming this from being a public field to being a private field, so your external function can access it; doesn't seem right to me